src/app/api/[[...route]]/routes/sharehouse.route.ts

```diff
@@ -16,6 +16,31 @@ import { InitialAssignedData } from '@/services/InitialAssignedData';
 import { sendEmail } from '@/lib/resend';
 import { EMAILS } from '@/constants/emails';
 
+interface IShareHouseWithOtherTables {
+  RotationAssignment: {
+    categories: {
+      tasks: {
+        id: string;
+        title: string;
+        description: string;
+        createdAt: Date;
+      }[];
+      id: string;
+      name: string;
+      createdAt: Date;
+    }[];
+    tenantPlaceholders: {
+      tenant: {
+        id: string;
+        name: string;
+        email: string;
+        createdAt: Date;
+      } | null;
+    }[];
+    rotationCycle: number;
+  } | null;
+}
+
```

Comment:
Could you move this to `/src/types/server.ts`?
Also, could you change the writing style to something like as following: https://github.com/Tascurator/tascurator-frontend/blob/5273ed95d352a327f50bbff950b212b75db3699a/src/types/server.ts#L115

Reply:
I fixed the tyoe.